media/js/firefox/os/partner_data.js

```diff
@@ -10,112 +10,97 @@ if (typeof Mozilla === 'undefined') {
-        },
-        'intex_cloudfx': {
-            'type': 'smartphone',
-            'display': 'Intex Cloud FX',
-            'countries': []
-        },
-        'lg_fireweb': {
-            'type': 'smartphone',
-            'display': 'LG Fireweb',
-            'countries': []
-        },
-        'orange_klif': {
-            'type': 'smartphone',
-            'display': 'Orange Klif',
-            'countries': ['ci', 'cm', 'eg', 'ke', 'mg', 'mu', 'ro', 'sn', 'tn']
-        },
-        'spice_fireonemifx1': {
-            'type': 'smartphone',
-            'display': 'Spice Fire One MI FX1',
-            'countries': []
-        },
-        'spice_fireonemifx2': {
-            'type': 'smartphone',
-            'display': 'Spice Fire One MI FX2',
-            'countries': ['in']
-        },
-        'symphony_gofoxf15': {
-            'type': 'smartphone',
-            'display': 'Symphony GoFox F15',
-            'countries': []
-        },
-        'zen_105fire': {
-            'type': 'smartphone',
-            'display': 'Zen 105 Fire',
-            'countries': []
-        },
-        'zte_open': {
-            'type': 'smartphone',
-            'display': 'ZTE Open',
-            'countries': ['mx']
-        },
-        'zte_openc': {
-            'type': 'smartphone',
-            'display': 'ZTE Open C',
-            'countries': ['de', 'fr', 'lu', 'uy']
-        },
-        'zte_openc2': {
-            'type': 'smartphone',
-            'display': 'ZTE Open C2',
-            'countries': ['pe']
-        },
-        'zte_open2': {
-            'type': 'smartphone',
-            'display': 'ZTE Open II',
-            'countries': ['ar', 'co', 'gt', 'ni', 'pa', 'pe', 'sv', 'uy']
+        'alcatel_onetouchfire' : {
+            'type' : 'smartphone',
+            'display' : 'Alcatel One Touch Fire',
+            'countries' : [ 'cl', 'co', 'mx' ]
+        },
+        'alcatel_onetouchfirec' : {
+            'type' : 'smartphone',
+            'display' : 'Alcatel One Touch Fire C',
+            'countries' : [ 'cl', 'co', 'gr', 'mx', 'ni', 'pe', 'uy' ]
+        },
+        'alcatel_onetouchfirec2g' : {
+            'type' : 'smartphone',
+            'display' : 'Alcatel One Touch Fire C 2G',
+            'countries' : []
+        },
+        'alcatel_onetouchfiree' : {
+            'type' : 'smartphone',
+            'display' : 'Alcatel One Touch Fire E',
+            'countries' : [ 'cz', 'ru' ]
+        },
+        'au_fx0' :
+        {'type' : 'smartphone', 'display' : 'au Fx0', 'countries' : [ 'jp' ]},
+        'cherrymobile_ace' : {
+            'type' : 'smartphone',
+            'display' : 'Cherry Mobile Ace',
+            'countries' : []
+        },
+        'huawei_y300-2' : {
+            'type' : 'smartphone',
+            'display' : 'Huawei Y300II',
+            'countries' : [ 'mx' ]
+        },
+        'intex_cloudfx' :
+        {'type' : 'smartphone', 'display' : 'Intex Cloud FX', 'countries' : []},
```

Comment:
@benjaminabel can you please fix these indentation changes too? I don't think this makes things more readable. Thanks